custom/_legacy/pact/templates/pact/patient/pactpatient_providers.html

```diff
@@ -240,7 +240,7 @@
                 self.is_loading(true);
                 self.showSave(false);
                 var arr = self.selected_providers();
-                var csrftoken = $.cookie('csrftoken');
+                var csrftoken = $("#csrfTokenContainer").val();
```

Comment:
I haven't checked for rest of the templates that this is changed, but this template extends `"reports/async/default.html"` which doesn't extend from `style/base.html`, so will it not miss the element `csrfTokenContainer `?